cg/services/post_processing/pacbio/data_transfer_service/utils.py

```diff
@@ -0,0 +1,63 @@
+from cg.constants.devices import DeviceType
+from cg.services.post_processing.pacbio.data_transfer_service.dto import (
+    PacBioSampleSequencingMetricsDTO,
+    PacBioSequencingRunDTO,
+    PacBioSMRTCellDTO,
+)
+from cg.services.post_processing.pacbio.metrics_parser.models import PacBioMetrics
+
+
+def get_smrt_cell_dto(metrics: PacBioMetrics) -> PacBioSMRTCellDTO:
+    internal_id: str = metrics.dataset_metrics.cell_id
+    return PacBioSMRTCellDTO(type=DeviceType.PACBIO, internal_id=internal_id)
+
+
+def get_sequencing_run_dto(metrics: PacBioMetrics) -> PacBioSequencingRunDTO:
+    hifi_mean_read_quality: str = f"Q{metrics.read.hifi_median_read_quality}"
+    return PacBioSequencingRunDTO(
+        type=DeviceType.PACBIO,
```

Comment:
Feel free to disregard but feels off that we would need to specify this in a DTO object whose name already contains "PacBio"

Reply:
This is part of the DTO functionality as this is one of the attributes we want to parse to the db for this instance.